test/test_ops.py

```diff
@@ -102,7 +102,13 @@ def _check_helper(self, device, dtype, op, variant, check):
         if not op.supports_dtype(dtype, torch.device(device).type):
             self.skipTest(f"Skipped! {op.name} does not support dtype {str(dtype)}")
 
-        samples = op.sample_inputs(device, dtype, requires_grad=True)
+        def is_inplace(variant):
+            if hasattr(variant, "__wrapped__"):
+                return variant.__wrapped__ is op.get_inplace()
+            return variant is op.get_inplace()
+
+        samples = op.sample_inputs(device, dtype, requires_grad=True,
```

Comment:
I think your proposal to wrap this in a try/except so not every sample_input needs to be updated to use **kwargs when this lands is a good idea.